crates/dojo-lang/src/print.rs

```diff
@@ -40,3 +40,42 @@ pub fn derive_print(db: &dyn SyntaxGroup, struct_ast: ItemStruct) -> RewriteNode
         ]),
     )
 }
+
+/// Derives PrintTrait for an enum.
+/// Parameters:
+/// * db: The semantic database.
+/// * enum_ast: The AST of the model enum.
+/// Returns:
+/// * A RewriteNode containing the generated code.
+pub fn handle_print_enum(db: &dyn SyntaxGroup, enum_ast: ItemEnum) -> RewriteNode {
+    let enum_name = enum_ast.name(db).text(db);
+    let prints: Vec<_> = enum_ast
+        .variants(db)
+        .elements(db)
+        .iter()
+        .map(|m| {
+            format!(
+                "{}::{}(value) => {{ debug::PrintTrait::print('{}'); \
+                 debug::PrintTrait::print(value); }}",
+                enum_name,
+                m.name(db).text(db).to_string(),
+                m.name(db).text(db).to_string()
+            )
+        })
+        .collect();
+
+    RewriteNode::interpolate_patched(
+        "#[cfg(test)]
+            impl $type_name$EnumPrintImpl of debug::PrintTrait<$type_name$> {
+                fn print(self: $type_name$) {
+                    match self {
+                        $print$
+                    }
+                }
+            }",
+        &UnorderedHashMap::from([
+            ("type_name".to_string(), RewriteNode::new_trimmed(enum_ast.name(db).as_syntax_node())),
+            ("print".to_string(), RewriteNode::Text(prints.join("\n"))),
```

Comment:
```suggestion
            ("print".to_string(), RewriteNode::Text(prints.join(",\n"))),
```

As we are in a match, cairo does require a trailing coma for each arm.